pkg/ibmcsidriver/config_watcher.go

```diff
@@ -72,22 +71,17 @@ func (cw *ConfigWatcher) updateSubnetList(oldObj, newObj interface{}) {
 		// Env variable VPC_SUBNET_IDS will be updated only when there is
 		// non empty data and there is change in configmap
 		if newConfig != "" && (newConfig != oldConfig) {
-			VPC_SUBNET_IDS := newData.Data[ConfigmapDataKey]
-			err := os.Setenv("VPC_SUBNET_IDS", VPC_SUBNET_IDS)
+			err := os.Setenv("VPC_SUBNET_IDS", newConfig)
 			if err != nil {
-				cw.logger.Warn("Config watcher is unable to update the subnet list..", zap.Any("Updated list", VPC_SUBNET_IDS), zap.Error(err))
+				cw.logger.Warn("Config watcher is unable to update the subnet list..", zap.Any("Updated list", newConfig), zap.Error(err))
```

Comment:
This logging is wrong, easily misunderstood.
Could you please correct it, it was a miss from my side.